### PR TITLE
Update for snippet

### PR DESCRIPTION
**Pull request by aallfredo**
@mcguish Could you review this and test locally make sure i'm not missing anything. This stuff scares me since it auto updates our clients.

### CODE REVIEW

File: TestHost/TestHost/snippet.js [hunk 1]
```diff
@@ -121,18 +133,24 @@ if (typeof Georiot == 'undefined') //The shield to avoid running the code twice
...
-                if (linkType == "apple") {
-                    currentLink.href = "http://target.georiot.com/Proxy.ashx?TSID=" + tsid + "&GR_URL=" + encodeURIComponent(currentLink.href);
-                } else if (linkType == "linkshare" || linkType == "tradedoubler" || linkType == "dgmperf") {
-                    var itunesUrl = Georiot.extractItunesLinkFromAffiliateUrl(currentLink, linkType);
-
-                    if (itunesUrl != "") {
-                        currentLink.href = "http://target.georiot.com/Proxy.ashx?TSID=" + tsid + "&GR_URL=" + itunesUrl;
-                    }
-                } else continue;
+                if (currentLink.hasAttribute('georiot-ignore') && currentLink.getAttribute('georiot-ignore') == 'true')
```
Comment (mcguish) [inline TestHost/TestHost/snippet.js]: you are moving the currentLink definition down to the else statement, so this doesn't seem like it would work

Comment (aallfredo) [inline TestHost/TestHost/snippet.js]: We don't want it to do anything if the georiot-ignore tag is there. Basically it avoids toughing that link

Comment (mcguish) [inline TestHost/TestHost/snippet.js]: but isn't if(currentLink.hasAttribute('georiot-ignore')... going to throw an undefined exception?

**Comment by mcguish** [comment]
@aallfredo looks good as long as the issue above isn't throwing an exception.  Test went fine as well.

File: TestHost/TestHost/snippet.js (hunk 1 above)
Comment (aallfredo) [inline TestHost/TestHost/snippet.js]: got it let me fix it just in case

**Pull request merged**